README.md

```diff
@@ -1 +1,237 @@
-# Vocdoni SDK
\ No newline at end of file
+# Vocdoni SDK
+
+The Vocdoni SDK is the easiest way to use [our new API][Vocdoni API], allowing
+you to easily create new voting processes, fetch their information and vote them.
+
+## Prerequisites
+
+Obviously you'll need a working nodejs environment, but other than that, you're
+free to use any package manager (either npm, yarn, pnpm...). Let's start by
+adding the SDK to your project:
+
+~~~bash
+# with yarn
+yarn add vocdoni-sdk
+# with npm
+npm i vocdoni-sdk
+# with pnpm
+pnpm add vocdoni-sdk
+~~~
+
+In case you want to create processes or vote to them, you'll need a signer to
```

Comment:
For creating elections or vote on them, blockchain transactions need to be build, thus a signer is required. Any kind of standard [ethers] signer should work.